SPECS-SIGNED/fwctl-signed/fwctl-signed.spec

```diff
@@ -0,0 +1,142 @@
+# OpenIB.org BSD license below:
+#
+#     Redistribution and use in source and binary forms, with or
+#     without modification, are permitted provided that the following
+#     conditions are met:
+#
+#      - Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      - Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials
+#        provided with the distribution.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+# BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+%{!?_name: %define _name fwctl}
+%{!?_version: %define _version 24.10}
+%{!?_release: %define _release OFED.24.10.0.6.7.1}
+
+%global target_kernel_version_full %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-headers))
+%global target_azurelinux_build_kernel_version %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}' $(/bin/rpm -q --whatprovides kernel-headers))
+%global target_kernel_release %(/bin/rpm -q --queryformat '%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-headers) | /bin/cut -d . -f 1)
+
+%global KVERSION %{target_kernel_version_full}
+%global K_SRC /lib/modules/%{target_kernel_version_full}/build
+
+# KMP is disabled by default
+%{!?KMP: %global KMP 0}
+
+%{!?KVERSION: %global KVERSION %{target_kernel_version_full}}
+%global kernel_version %{KVERSION}
+%global krelver %(echo -n %{KVERSION} | sed -e 's/-/_/g')
+# take path to kernel sources if provided, otherwise look in default location (for non KMP rpms).
+%{!?K_SRC: %global K_SRC /lib/modules/%{KVERSION}/build}
+
+# define release version
+%{!?src_release: %global src_release %{_release}_%{krelver}}
+%if "%{KMP}" != "1"
+%global _release1 %{src_release}
+%else
+%global _release1 %{_release}
+%endif
+%global _kmp_rel %{_release1}%{?_kmp_build_num}%{?_dist}
+
+Summary:	 %{_name} Driver
+Name:		 %{_name}
+Version:	 %{_version}
+Release:	 1%{?dist}
+License:	 GPLv2
+Url:		 http://nvidia.com
+Group:		 System Environment/Base
+
+# This package's "version" and "release" must reflect the unsigned version that
+# was signed.
+# An important consequence is that when making a change to this package, the
+# unsigned version/release must be increased to keep the two versions consistent.
+# Ideally though, this spec will not change much or at all, so the version will
+# just track the unsigned package's version/release.
```

Comment:
No need for the comment - the specs entanglement PR check will handle that for you. Just make sure to update the set of spec pairs, which need to be kept in sync.

Reply:
Removed.